docs/command_context.rst

```diff
@@ -72,3 +68,10 @@ Context variables only available to Jobs
     The last successful run date (defaults to current date if there was no
     previous successful run). Supports date arithmetic using the form
     ``{last_success#shortdate-1}``.
+
+**manual**
+    ``true`` if the job was run manually. ``false`` otherwise.
+
+**namespace**
+    The namespace of the config where the job comes from. Often ``MASTER`` or ``servicename``.
```

Comment:
Please give an example here, so if a job "bar" is defined in for the foo service in foo/tron-norcal-prod.yaml, then the job would appear in Tron as "foo.bar", where `foo` is the namespace.